src/sentry/utils/performance_issues/base.py

```diff
@@ -175,11 +175,12 @@ def get_url_from_span(span: Span) -> str:
     return url
 
 
-def fingerprint_spans(spans: List[Span]):
+def fingerprint_spans(spans: List[Span], unique_only: bool = False):
```

Comment:
Do the other detectors using `fingerprint_spans` need `unique_only=False`? Or are you doing this to limit impact from fingerprint changes for existing? If `true` is generally correct (I suspect it is), should we be fixing other detectors? I'd ask whoever owns the other ones to check.

Reply:
yeah mainly to limit fingerprint impact, I also suspect that `True` is generally correct.

I'll look into which other detectors use it and check separate to this PR.

Good call 👍